xmtp_id/src/associations/state.rs

```diff
@@ -127,6 +127,15 @@ impl AssociationState {
     }
 }
 
+impl From<AssociationState> for AssociationStateDiff {
+    fn from(state: AssociationState) -> Self {
+        AssociationStateDiff {
+            new_members: state.members.keys().cloned().collect(),
+            removed_members: vec![],
+        }
+    }
+}
```

Comment:
When would we need this one?

Reply:
I actually borrowed it from Andrew's PR. You need it when you want to work with a diff, but there is only one entry in the list.

I do think @37ng is right that it's kinda confusing to use a `From` to do this.

Reply:
The validation service is using it for the same purpose

Reply:
`as_diff` definitely clearer. Just needed a way to make a `Default` diff in case we only had one update for a diff.